### PR TITLE
 Implement basic crud for Products and Categories

### PR DESCRIPTION
**Pull request by songofroland**
resolved Basic adding of new products

### CODE REVIEW

File: easywarehouse/products/forms.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+from django.forms import ModelForm
+
+from . import models
+
+
+class ProductForm(ModelForm):
+    class Meta:
+        model = models.Product
+        fields = ["category", "name", "tags"]  # TODO: Split tags to an array
```
[the remark this answers is not in the document]
Comment (wilmhit) [reply easywarehouse/products/forms.py:9]: But this is model form so no postgres here. Idk wdym

**Review by wilmhit — CHANGES_REQUESTED** [review]

File: easywarehouse/templates/products/details.html [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<h1>{{ object.name }}</h1>
+<p>{{ object.public_id }}</p>
+<a href="{% url "products-delete" object.id %}">Delete the product</a>
```
[the remark this answers is not in the document]
Comment (songofroland) [reply easywarehouse/templates/products/details.html:3]: Yup, I forgot the black does not check html

File: easywarehouse/products/urls.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+from django.urls import path
+
+from . import views
+
+urlpatterns = [
+    path("", views.index),
+    path("dashboard", views.dashboard),
+    path("products", views.ListProducts.as_view(), name="products-list"),
+    path("products/add/", views.AddProduct.as_view(), name="products-add"),
+    path("products/<int:pk>/delete", views.DeleteProduct.as_view(), name="products-delete"),
```
[the remark this answers is not in the document]
Comment (songofroland) [reply easywarehouse/products/urls.py:10]: It's a numerical id, not the name. Anyway, I'll change it to `id`

[the remark this answers is not in the document]
Comment (songofroland) [reply easywarehouse/products/urls.py:10]: Update: it must stay `pk`, it's required by django

**Review by wilmhit — APPROVED** [review]

**Pull request merged**